package.json

```
@@ -45,7 +45,7 @@
     "postcss-discard-comments": "^7.0.3",
     "postcss-nesting": "^13.0.1",
     "prettier": "^3.4.2",
-    "puppeteer": "23.3.1",
+    "puppeteer": "23.10.2",
```

Can we unlock the version again, as discussed in PR #19115?
```suggestion
    "puppeteer": "^23.10.2",
```